RELEASE_PROCESS.md

```diff
@@ -70,7 +70,7 @@ Umbrella Testing Step-by-Step:
 1. Optionally, fill-in the 'Message' field with something descriptive.
 1. Select 'Create Build'.
 
-Currently (02/21), the Umbrella pipeline does not perform a test login to Chef Manage, so this should be done manually.  One method is to run representative AWS and Azure Umbrella scenarios locally to get running server instances to test.
+Currently (02/21), the Umbrella pipeline does not perform a test login to Chef Manage, so this should be done manually.  One method is to run representative AWS and Azure Umbrella scenarios locally to get running server instances to test.Chef Manage should be verified on IPV4 and IPV6 setup.
```

Comment:
Thanks for adding this! It would also be helpful to mention that `chef-server-ctl install chef-manage` on the local setup will install the latest stable release of chef-manage. Where as umbrella if a version for chef-manage is not defined will pick the latest current (unstable) version.